src/components/access-code/_macro.njk

```diff
@@ -28,7 +28,7 @@
 
     {% if params.error is not defined %}
         {% call onsPanel({
-                "classes": "ons-u-mb-s"
+                "classes": "ons-u-mb-s" + (' ' + params.classes if params.classes is defined and params.classes)
```

Comment:
The ' ' should be inside the if otherwise it would leave whitespace in the class chain.

Reply:
This doesn't allow for changing the margin util class, so I'll change this to use that class as default.

Reply:
Doesn't appear to add whitespace...

![Screenshot 2022-06-16 at 11 44 55](https://user-images.githubusercontent.com/43346934/174055594-7918b87b-0beb-48fc-8b42-baaa0d207217.png)